src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transorm $trstn-drtn,
```

Comment:
```suggestion
    transition: transform $trstn-drtn,
```
Should fix why transition not applied

Reply:
what is the strange name `$trstn-drtn`?)

Reply:
thank you for the corrections,
mistakes have been fixed

Reply:
yep, name should be always understandable